protected/humhub/components/ModuleManager.php

```diff
@@ -364,25 +342,25 @@ public function getEnabledModules($options = [])
      */
     public function hasModule($id)
     {
-        return (array_key_exists($id, $this->modules));
+        return ($module = $this->modules[$id] ?? null) && !$module->isMissing && $module->getClassExists();
     }
 
     /**
-     * Returns weather or not the given module id belongs to an core module.
+     * Returns weather or not the given module id belongs to a core module.
```

Comment:
Can this be fixed to say `whether` as currently this is the wrong word used, I'm surprised that this has gone unchecked for so long.